src/code/circuit_python/hello.md

```diff
@@ -0,0 +1,179 @@
+# Hello World
```

Comment:
I like the overall flow of this lesson, but it is written without any direction from the top.  There needs to be some statement at the introduction that the outcome of the lesson will be learning to blink the LED on and off, so that the modules you introduce have purpose.